addons/dragging/functions/fnc_createClone.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut, johnb43
+ * Creates a draggable / carryable clone of a dead unit.
+ *
+ * Arguments:
+ * 0: Dead unit <OBJECT>
+ *
+ * Return Value:
+ * Cloned unit <OBJECT>
+ *
+ * Example:
+ * [player] call ace_dragging_fnc_createClone;
+ *
+ * Public: No
+ */
+params ["_target"];
+
+private _clone = QGVAR(clone) createVehicle [0, 0, 0];
+
+// Clone loadout
+_clone setUnitLoadout getUnitLoadout _target;
```

Comment:
I think this will cause problems with acre or other systems expecting unique ids

Reply:
> I think this will cause problems with acre or other systems expecting unique ids

Filter them out using `acre_api_fnc_filterUnitLoadout` and whatever TFAR has respectively wouldn't suffice?

Reply:
Just use CBA's `CBA_fnc_getLoadout`.